lisdf/components/model.py

```diff
@@ -70,7 +54,13 @@ def quat_xyzw(self) -> Vector4f:
 
     @cached_property
     def rpy(self) -> Vector3f:
-        return euler_from_quaternion(self.quat_xyzw)
+        return euler_from_quaternion(self.quat_xyzw)  # type: ignore
+
+    def to_sdf(self) -> str:
+        return (
+            f"<pose>{self.pos[0]}, {self.pos[1]}, {self.pos[2]}, "
+            "{self.rpy[0]}, {self.rpy[1]}, {self.rpy[2]}</pose>"
```

Comment:
You need to make the second line a f-string or the formatting won't work

Reply:
Good catch!